src/main/java/bot/assembly/function/BotTaskStatusGeneratorUnit.java

```diff
@@ -0,0 +1,74 @@
+package bot.assembly.function;
+
+import bot.assembly.memory.BotStaticMemoryUnit;
+import bot.assembly.memory.BotDynamicMemoryUnit;
+import bot.assembly.error.EmptyTaskListException;
+
+import bot.assembly.task.Task;
+
+import java.util.List;
```

Comment:
Should the order and arrangement of the imports be this way (similar to the other imports in your code)?